packages/@react-native-windows/cli/src/runWindows/utils/msbuildtools.ts

```diff
@@ -97,8 +97,10 @@ export default class MSBuildTools {
       args.push('/maxCpuCount');
     }
 
-    if (target) {
-      args.push(`/t:${target}`);
+    if (target === 'build') {
+      args.push('/restore', '/p:RestorePackagesConfig=true');
```

Comment:
can you update the vswhere logic to ensure VS version is 16.5 and add an error if not?

Also we should update rnw-dependencies.ps1 to ensure VS is >16.5